record-and-playback/core/scripts/bigbluebutton.yml

```diff
@@ -7,6 +7,14 @@ raw_screenshare_src: /usr/share/red5/webapps/screenshare/streams
 raw_webrtc_deskshare_src: /usr/share/red5/webapps/video-broadcast/streams
 raw_deskshare_src: /var/bigbluebutton/deskshare
 raw_presentation_src: /var/bigbluebutton
+note_endpoint: http://localhost:9001/p
+note_formats:
```

Comment:
I'm not sure whether or not we want to download all of the ethernote formats. I can justify three of them:

* etherpad - this contains timing information, and might be useful for recordings if we want to play back synchronized updates
* html - I assume this is preformatted flat html file? Could be useful for simple recording formats that just include it in an iframe, no further processing needed.
* pdf - If we want to offer a download of the final shared note state, having preconverted pdf is convenient

The other formats I think we're unlikely to use. If we really need them later we can do the conversion ourselves from html or re-enable them.